doc/espresso-dev-node.md

```diff
@@ -0,0 +1,69 @@
+| `account_index`                 | `u32`           | `ESPRESSO_DEPLOYER_ACCOUNT_INDEX`    | Account index in the L1 wallet generated by `MNEMONIC` to use when deploying the contracts.                                                                             |
+| `sequencer_api_port`            | `u16`           | `ESPRESSO_SEQUENCER_API_PORT`        | Port that the HTTP API will use.                                                                                                                                        |
+| `sequencer_api_max_connections` | `Option<usize>` | `ESPRESSO_SEQUENCER_MAX_CONNECTIONS` | Maximum concurrent connections allowed by the HTTP API server.                                                                                                          |
+| `builder_port`                  | `Option<u16>`   | `ESPRESSO_BUILDER_PORT`              | Port for connecting to the builder. If this is not provided, an available port will be selected.                                                                        |
+| `prover_port`                   | `Option<u16>`   | `ESPRESSO_PROVER_PORT`               | Port for connecting to the prover. If this is not provided, an available port will be selected.                                                                         |
+| `dev_node_port`                 | `u16`           | `ESPRESSO_DEV_NODE_PORT`             | Port for the dev node. This is used to provide tools and information to facilitate developers debugging. If this is not provided, will use `20000` as the default port. |
+
+## APIs
+
+Once you have successfully run the dev node, you can access the corresponding ports to call the APIs of the
+[`builder`](https://docs.espressosys.com/sequencer/api-reference/builder-api),
+[`sequencer`](https://docs.espressosys.com/sequencer/api-reference/sequencer-api), and `prover`.
+
+In addition, you can access the `dev_node_port` to retrieve debugging information. Here are the details of the dev node
+API.
+
+### GET /api/dev-info
+
+This endpoint returns some debug information for you.
+
+An example response is like this:
+
+```json
+{
+  "builder_url": "http://localhost:41003/",
+  "prover_port": 23156,
+  "l1_url": "http://localhost:8545/",
+  "light_client_address": "0xb075b82c7a23e0994df4793422a1f03dbcf9136f"
+}
+```
+
+### POST /api/set-hotshot-down
+
+This endpoint sets the hotshot down at the given L1 height.
+
+By calling this, the L1 height in the light contract will be frozen, and rollups will detect the HotShot failure. This
+is intended for testing rollups' functionalities when HotShot is down.
+
+Parameters
+
+| Name   | Type    | Description                              |
+| ------ | ------- | ---------------------------------------- |
+| height | integer | The L1 height from which hotshot is down |
+
+### POST /api/set-hotshot-up
+
+This endpoint sets the hotshot up in the light client contract.
```

Comment:
Similarly, let's be more precise and avoid using the terms "hotshut up/down" in the documentation.